.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 /build/
+.idea/
```

Comment:
You could use a global gitignore. It would save you a lot of troubles and time for each repo. See https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore and the linked recommended global gitignore.